### ISSUE TITLE
Defaults for fragments/views and the different androidx widgets (as library module, not sample)

### ISSUE THREAD

**Issue by Zhuinden** [Zhuinden/simple-stack#126]
Once I actually do modularization (and the switch to `androidx`), I should add modules that help with the initial integration of the library.

Reading through samples is not *that* intuitiv, especially when contrasted against the competition.

-----

Things that should be handled:

- navigation drawer

- bottom nav view (with multi-stack support :scream: (edit: or not, material design says it's not needed anymore either lol))

- possibly shared element transitions :thinking: 

Basically everything that https://developer.android.com/reference/androidx/navigation/ui/NavigationUI can do.

**Comment by Zhuinden** [comment]
Too limiting. `NavigationUI` just isn't really worth the time in this regard to mirror.

**Comment by Zhuinden** [comment]
But I should definitely put in some drawer layout samples.
